flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql2rel/AuxiliaryConverter.java

```diff
@@ -53,25 +53,6 @@ public Impl(SqlFunction f) {
 		public RexNode convert(RexBuilder rexBuilder, RexNode groupCall,
 			RexNode e) {
 			return rexBuilder.makeCall(this.f, e);
-			// FLINK QUICK FIX
-			// we do not use this logic right now
-//      switch (f.getKind()) {
-//      case TUMBLE_START:
-//      case HOP_START:
-//      case SESSION_START:
-//      case SESSION_END: // TODO: ?
-//        return e;
-//      case TUMBLE_END:
-//        return rexBuilder.makeCall(
-//            SqlStdOperatorTable.PLUS, e,
-//            ((RexCall) groupCall).operands.get(1));
-//      case HOP_END:
-//        return rexBuilder.makeCall(
-//            SqlStdOperatorTable.PLUS, e,
-//            ((RexCall) groupCall).operands.get(2));
-//      default:
```

Comment:
Let's just keep this comments.

Reply:
good point. I should kept the comments. thanks for reminder.